src/lib/isTezosAddress.js

```diff
@@ -0,0 +1,8 @@
+import assertString from './util/assertString';
+
+const xtz = /^(tz1|KT1)[0-9a-z]{33}$/i;
```

Comment:
this regex will fail -> cater for uppercase chars

```/^(tz1|KT1)[0-9a-zA-Z]{33}$/i;```

Reply:
the `i` modifier at the end of the regex makes it case insensitive so it does work.

having said that, on a second thought making it case insensitive is actually not a good idea here, as it allows a lowercase 'kt1' prefix, which is not a valid Tezos address.

I'll fix this. thanks.

Reply:
Correct regex: `/^(tz1|KT1)[0-9a-zA-Z]{33}$/;`